client/src/components/NavBar/NavBar.tsx

```diff
@@ -76,15 +76,15 @@ export function Navbar(props: any) {
       "ChIJn58N1B9gUocRpAXOXPbFcOo"
     );
 
-    // console.log(getRestaurantReviewsByRestaurant);
-    // console.log(getRestaurantReviewsByUser);
-    // console.log(getRestaurantReview);
-    // console.log(deleteRestaurantReview);
-    // console.log(updateRestaurantReview);
-    // console.log(createRestaurantReview);
-    // console.log(getRestaurantsResults);
-    // console.log(getRestaurantResults);
-    // console.log(saveRestaurantResults);
+    console.log(getRestaurantReviewsByRestaurant);
+    console.log(getRestaurantReviewsByUser);
+    console.log(getRestaurantReview);
+    console.log(deleteRestaurantReview);
+    console.log(updateRestaurantReview);
+    console.log(createRestaurantReview);
+    console.log(getRestaurantsResults);
+    console.log(getRestaurantResults);
+    console.log(saveRestaurantResults);
```

Comment:
We should always keep these lines commented out when pushing to your branch, they are just for local testing.